src/unix/freebsd.c

```diff
@@ -266,7 +266,10 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
 
 int uv__sendmmsg(int fd, struct uv__mmsghdr* mmsg, unsigned int vlen) {
 #if __FreeBSD__ >= 11
-  return sendmmsg(fd, mmsg, vlen, /* flags */ 0);
+  return sendmmsg(fd,
+                  (struct mmsghdr*) mmsg,
```

Comment:
```suggestion
                  &mmsg->msg_hdr,
```

Reply:
@vtjnash `&mmsg->msg_hdr` is of type `struct msghdr*` which is accepted by `sendmsg` but `sendmmsg` requires a `struct mmsghdr*` (notice the second `m`). So, this will reintroduce the warning I think.